src/helpers/tagHelpers.js

```diff
@@ -0,0 +1,35 @@
+import Models from '../db/models';
+
+const { Tag } = Models;
+
+/**
+ * @description class representing Tag Helpers
+ * @class TagHelpers
+ */
+class TagHelpers {
+  /**
+   * @description - This method is responsible for adding all the new tags
+   * @static
+   * @param {object} newArticleTags - Array of tags(new or old) to be added
+   * @returns {object} - an array of tag IDs
+   * @memberof TagHelpers
+   */
+  static async addNewTag(newArticleTags) {
+    try {
+      const tagsNeeded = [];
+      const allTags = await Promise.all(newArticleTags.map(tag => Tag.findOrCreate({
```

Comment:
Nice job @oluwajuwon, but can you explain why you are using findOrCreate.

Reply:
Nice question @okoroemeka Since our tags are unique, I have to first find the tags if they have been created before so that it doesn't create a new one, and if they haven't it should create it.